src/Fhir.Anonymizer.Tool/Stu3-config-draft.json

```diff
@@ -0,0 +1,267 @@
+    {"path": "Coverage.Group.subGroup", "method": "redact"},   ///Newadd
+    {"path": "Coverage.Group.plan", "method": "redact"},   ///Newadd
+    {"path": "Coverage.Group.subPlan", "method": "redact"},   ///Newadd
+    {"path": "Coverage.Group.class", "method": "redact"},   ///Newadd
+    {"path": "Coverage.Group.subClass", "method": "redact"},   ///Newadd
+    {"path": "Coverage.Group.groupDisplay", "method": "redact"},  ///Newadd
+    {"path": "Coverage.Group.subGroupDisplay", "method": "redact"},   ///Newadd
+    {"path": "Coverage.Group.planDisplay", "method": "redact"},   ///Newadd
+    {"path": "Coverage.Group.subPlanDisplay", "method": "redact"},   ///Newadd
+    {"path": "Coverage.Group.classDisplay", "method": "redact"},   ///Newadd
+    {"path": "Coverage.Group.subClassDisplay", "method": "redact"},   ///Newadd
+    //{"path": "CoverageEligibilityRequest.insurance.businessArrangement", "method": "redact"},
+    //{"path": "CoverageEligibilityResponse.disposition", "method": "redact"},
+    //{"path": "CoverageEligibilityResponse.insurance.item.name", "method": "redact"},
+    //{"path": "CoverageEligibilityResponse.insurance.item.description", "method": "redact"},
+    //{"path": "CoverageEligibilityResponse.insurance.item.benefit.allowed", "method": "redact"},
+    //{"path": "CoverageEligibilityResponse.insurance.item.benefit.used", "method": "redact"},
+    //{"path": "CoverageEligibilityResponse.preAuthRef", "method": "redact"},
+    {"path": "EligibilityRequest.businessArrangement", "method": "redact"}, ///Newadd
+    {"path": "EligibilityResponse.disposition", "method": "redact"},   ///Newadd
+    {"path": "EligibilityResponse.insurance.benefitBalance.name", "method": "redact"},   ///Newadd
+    {"path": "EligibilityResponse.insurance.benefitBalance.description", "method": "redact"},   ///Newadd 
+    {"path": "EligibilityResponse.insurance.benefitBalance.financial.allowed", "method": "redact"},   ///Newadd
+    {"path": "EnrollmentResponse.disposition", "method": "redact"},
+    {"path": "Claim.supportingInfo.value", "method": "redact"},
+    {"path": "Claim.insurance.businessArrangement", "method": "redact"},
+    {"path": "Claim.insurance.preAuthRef", "method": "redact"},
+    {"path": "ClaimResponse.disposition", "method": "redact"},
+    {"path": "ClaimResponse.insurance.preAuthRef", "method": "redact"}, //Modified
+    {"path": "ClaimResponse.processNote.text", "method": "redact"},
+    {"path": "ClaimResponse.insurance.businessArrangement", "method": "redact"},
+    {"path": "Invoice.cancelledReason", "method": "redact"},
+    {"path": "PaymentReconciliation.disposition", "method": "redact"},
+    {"path": "PaymentReconciliation.processNote.text", "method": "redact"},
+    {"path": "Account.name", "method": "redact"},
+    {"path": "Account.description", "method": "redact"},
+    {"path": "ChargeItem.definition", "method": "redact"},   ///??????????????????

```

Comment:
We could keep this field?

Reply:
Agree. It is a uri and seems not sensitive?